spherogram_src/links/invariants.py

```diff
@@ -504,18 +502,15 @@ def goeritz_matrix(self, return_graph=False):
         G = self.white_graph()
         V = G.vertices()
         N = len(V)
-        m = matrix(N,N)
-        vertex = dict((v, n) for n, v in enumerate(V))
+        m = matrix(N, N)
+        vertex = {v: n for n, v in enumerate(V)}
         for e in G.edges(sort=False):
             i, j = vertex[e[0]], vertex[e[1]]
-            m[(i,j)] = m[(j,i)] = m[(i,j)] + e[2]['sign']
+            m[(i, j)] = m[(j, i)] = m[(i, j)] + e[2]['sign']
         for i in range(N):
-            m[(i,i)] = -sum(m.column(i))
+            m[(i, i)] = -sum(m.column(i))
         m = m.delete_rows([0]).delete_columns([0])
-        if return_graph:
-            return m, G
-        else:
-            return m
+        return m, G if return_graph else m
```

Comment:
Line 513 is actually an error, since it amounts to `return m, (G if return_graph else m)` not `return (m, G) if return_graph else m)`.  I'm confused how this got past the CI since I got an error as soon as I merged it into main.